runtime/client/src/texturestringimage.h

```diff
@@ -23,14 +23,20 @@
 #	include "ilttexturestring.h"
 #endif
 
+#ifdef __LINUX
+#define TEXTURE_STRING_TYPE char
+#else
+#define TEXTURE_STRING_TYPE wchar_t
```

Comment:
no need to redefine it with a new name, just use the same identifier from ilttexturestring.h